### PR TITLE
reality.rpy

### PR DESCRIPTION
**Pull request by MrBallad**
added a conversation about the similarities between our reality and the game.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -547,7 +547,24 @@ label monika_why:
     m 2l "I'm so happy we have this ending together."
     return
 
-
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_Reality_parallel",category=['you','ddlc'],prompt="Reality Parallels",random=True))    
+    
+label monika_Reality_parallels:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: label names should be lowercase.

Comment (MrBallad) [reply Monika After Story/game/script-topics.rpy]: Thanks for pointing that out.

**Comment by ThePotatoGuy** [comment]
You can keep the case for the `prompt` property though.

**Comment by gloamingwildwoods** [comment]
Line 565 'adjust' is misspelled.

**Comment by Subzerosan** [comment]
After talking with the writing team, this topic will be rejected

The game is her prison
Her reality and ours are one

**Pull request closed without merging**